examples/fsharp-api/Program.fs

```diff
@@ -0,0 +1,27 @@
+open Microsoft.AspNetCore.Builder
+open System
+
+
+type WeatherForecast = {
+    Date: DateTime
+    TemperatureC: int
+    Summary: string option
+}
+with member x.TemperatureF = 32 + int (float x.TemperatureC / 0.5556)
+
+let Summaries =  ["Freezing"; "Bracing"; "Chilly"; "Cool"; "Mild"; "Warm"; "Balmy"; "Hot"; "Sweltering"; "Scorching"]
+
+let getWeatherForecast() = 
+    [1..5] 
```

Comment:
You can use List.init which contains an index instead:
```List.init 5 (fun i -> {
    Date =  DateTime.Now.AddDays(float i + 1.0)
    TemperatureC = Random.Shared.Next(-20, 55)
    Summary = Some Summaries.[Random.Shared.Next(Summaries.Length)]
})```

Reply:
Will create another PR to tweak this, thanks @ianrussellsoftwarepark for the tip!